public/static/locales/en/oauth.json

```diff
@@ -0,0 +1,15 @@
+{
+    "invalid_request": "The authorization request sent to the remote system by the DE was invalid. Please contact support.",
+    "unauthorized_client": "The DE is not authorized to request access to the remote system. Please contact support.",
+    "access_denied": "The remote system denied the authorization request. Please try again or contact support.",
+    "unsupported_response_type": "The remote system does not support the response requested by the DE. Please contact support.",
+    "invalid_scope": "The remote system does not support the scope requested by the DE. Please contact support.",
+    "server_error": "The remote system system encountered an error. Please try again or contact support.",
+    "temporarily_unavailable": "The remote system is temporarily unavailable. Please try again or contact support.",
```

Comment:
These error codes come directly from the OAuth specification. Having a unique message for each code will help us to troubleshoot these errors if they occur.